### ISSUE TITLE
Get-BcArtifactUrl fails on powershell7

### ISSUE THREAD

**Issue by gntpet** [microsoft/navcontainerhelper#3269]
**Describe the issue**
Get-BcArtifactUrl fails on pwsh7. It used to work last week. 

**Scripts used to create container and cause the issue**
```
Get-BcArtifactUrl -country nl -type OnPrem -version 11.0.49497.0  
```

**Full output of scripts**

> Error querying artifacts. Error message was Cannot convert value "﻿<?xml version="1.0" encoding="utf-8"?><EnumerationResults ContainerName="https://bcartifacts.blob.core.windows.net/onprem/"><Prefix>11.0.49497.0</Prefix><Blobs><Blob><Name>11.0.49497.0/at</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/at</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:34:48 GMT</Last-Modified><Etag>0x8DAD4487450DE70</Etag><Content-Length>187579288</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>xYLQz6psUP6MazoIksKliw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/au</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/au</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:35:08 GMT</Last-Modified><Etag>0x8DAD44880AD9166</Etag><Content-Length>188991039</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>RUIzaayYPPoK1vyTgugu9A==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/be</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/be</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:35:31 GMT</Last-Modified><Etag>0x8DAD4488E404D2B</Etag><Content-Length>207468258</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>KckGY4nl2cCdUjq+FAU2lw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/ca</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/ca</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:35:52 GMT</Last-Modified><Etag>0x8DAD4489AA1BA5F</Etag><Content-Length>203338647</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>fOpq/WTh8nXoeyZxrQiDNw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/ch</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/ch</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:36:20 GMT</Last-Modified><Etag>0x8DAD448AB2F6C63</Etag><Content-Length>230459843</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>hZ394kXSOMAwAryMFfsOrw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/cz</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/cz</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:36:41 GMT</Last-Modified><Etag>0x8DAD448B7CEC3D7</Etag><Content-Length>201108664</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>7/+vGLgxnD+cQ3O/Qbmqvg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/de</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/de</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:37:01 GMT</Last-Modified><Etag>0x8DAD448C3E43EEB</Etag><Content-Length>198035943</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>K/MeoZY110lp/cbC0U1EKw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/dech</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/dech</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:37:21 GMT</Last-Modified><Etag>0x8DAD448CFC93B36</Etag><Content-Length>191920485</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>oAT4LI1uMmdpjGqqtV9KgA==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/dk</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/dk</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:37:45 GMT</Last-Modified><Etag>0x8DAD448DDF9586A</Etag><Content-Length>201742732</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>mSUXjf8QGGrK4peaonq5yg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/es</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/es</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:38:06 GMT</Last-Modified><Etag>0x8DAD448EAA753BF</Etag><Content-Length>193050427</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>3vhqEI5MPCVFgFrlaUwYWQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/fi</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/fi</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:38:25 GMT</Last-Modified><Etag>0x8DAD448F5AA4E63</Etag><Content-Length>185780662</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>hPHTTXXWCX5EJ25GJLvjKg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/fr</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/fr</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:38:44 GMT</Last-Modified><Etag>0x8DAD44900F7187A</Etag><Content-Length>189680576</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>/rjvfXygfBLdbRWtDxoxuQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/frbe</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/frbe</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:39:03 GMT</Last-Modified><Etag>0x8DAD4490CAA368A</Etag><Content-Length>189934419</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>7wyAfIYV3A8VEwbw1iyAmQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/frca</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/frca</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:39:23 GMT</Last-Modified><Etag>0x8DAD44918A305F6</Etag><Content-Length>197761399</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>K/PA4DepPyXgq0ZVINQSfg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/frch</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/frch</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:39:43 GMT</Last-Modified><Etag>0x8DAD44924A98F12</Etag><Content-Length>193251805</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>tXEdCgxI+SXg5Aa6lZ/Baw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/gb</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/gb</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:40:04 GMT</Last-Modified><Etag>0x8DAD44930C8307C</Etag><Content-Length>192604201</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>vrci2iMRMfAGYLCCVEe8Pg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/is</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/is</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:40:23 GMT</Last-Modified><Etag>0x8DAD4493C1BFE62</Etag><Content-Length>186130783</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>feMNGCxW0ZBHOL+M3mW6Wg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/it</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/it</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:40:43 GMT</Last-Modified><Etag>0x8DAD449483326DE</Etag><Content-Length>189976052</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>BPjrkoAmKxf+35FtBT7MhA==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/itch</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/itch</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:41:04 GMT</Last-Modified><Etag>0x8DAD44954E825F4</Etag><Content-Length>193274245</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>OPUhkDbf5t30P2EOU5824A==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/mx</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/mx</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:41:24 GMT</Last-Modified><Etag>0x8DAD44960C6936F</Etag><Content-Length>197947463</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>N3tQ5uk6Hj/5eadwLEPVsQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/na</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/na</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:42:06 GMT</Last-Modified><Etag>0x8DAD449796FA7BD</Etag><Content-Length>320975618</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>Hxqvnw0Jp+k4x27sDnl/rw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/nl</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/nl</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:42:26 GMT</Last-Modified><Etag>0x8DAD4498582D921</Etag><Content-Length>195797996</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>s0IYnPHP0b8ifIibJJBfTw==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/nlbe</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/nlbe</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:42:46 GMT</Last-Modified><Etag>0x8DAD44991684A7D</Etag><Content-Length>188556961</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>pSaVJCR9Ki58fm6XcdNebg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/no</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/no</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:43:06 GMT</Last-Modified><Etag>0x8DAD4499D71924A</Etag><Content-Length>189126109</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>GJvh1zFBakVwot8uQHMF8g==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/nz</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/nz</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:43:28 GMT</Last-Modified><Etag>0x8DAD449AA5B0832</Etag><Content-Length>187345449</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>VtiSPFWjM79QN2nwesy5BQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/platform</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/platform</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:45:46 GMT</Last-Modified><Etag>0x8DAD449FCA64470</Etag><Content-Length>262835283</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>QYRvm8ezQOEZYwDxYy7v/A==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/ru</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/ru</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:43:51 GMT</Last-Modified><Etag>0x8DAD449B834FB93</Etag><Content-Length>219376126</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>jq7LS/2tyrnGFpVNezl98A==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/se</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/se</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:44:11 GMT</Last-Modified><Etag>0x8DAD449C41F001E</Etag><Content-Length>187335289</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>ceSroAXsFi4WS+GOlGa9Cg==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/us</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/us</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:44:32 GMT</Last-Modified><Etag>0x8DAD449D087E622</Etag><Content-Length>200183106</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>8AvcH+rsNBEUS6T6QYhYqQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob><Blob><Name>11.0.49497.0/w1</Name><Url>https://bcartifacts.blob.core.windows.net/onprem/11.0.49497.0/w1</Url><Properties><Last-Modified>Fri, 02 Dec 2022 09:45:56 GMT</Last-Modified><Etag>0x8DAD44A02B8194A</Etag><Content-Length>125627519</Content-Length><Content-Type>application/octet-stream</Content-Type><Content-Encoding /><Content-Language /><Content-MD5>d8q15Cz4KKqCkZLNINDtzQ==</Content-MD5><Cache-Control /><BlobType>BlockBlob</BlobType><LeaseStatus>unlocked</LeaseStatus></Properties></Blob></Blobs><NextMarker /></EnumerationResults>" to type "System.Xml.XmlDocument". Error: "The specified node cannot be inserted as the valid child of this node, because the specified node is the wrong type."
> 
> Repeating download attempt ( 2  of  10 )...
> 
> 

Best Regards, 
Gintautas

**Comment by eknraw** [comment]
Looks like it was resolved in bccontainerhelper 6.0.2.

https://github.com/microsoft/navcontainerhelper/issues/3259

**Comment by gntpet** [comment]
Thanks for the tip @eknraw , updated solved the problem
